website/content/docs/agent-and-proxy/agent/template.mdx

```diff
@@ -107,12 +107,17 @@ failures.
   This setting will not change how often Vault Agent Templating renders leased
   secrets. Uses [duration format strings](/vault/docs/concepts/duration-format).
 
+- `max_connections_per_host` `(int: 10)` - Limits the total number of connections
+  that Vault Agent Template can use for a particular Vault host. This limit
```

Comment:
```suggestion
  that the Vault Agent templating engine can use for a particular Vault host. This limit
```

Reply:
This seems to be consistent with how we talk about it elsewhere in the file, and I think it reads a little cleaner